index.html

```diff
@@ -6,17 +6,13 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="queries.css">
-  <script src="menu.js"></script>
   <title>Frank's Portfolio</title>
 </head>
 
 <body>
```

Comment:
[OPTIONAL] I think you could hide the `overflow` on the `body` tag when the user opens a modal. It's a nice little improvement in user experience. Although still subjective. Feel free to not add it if you like the functionality as it is.